notebooks/create_htmls.ipynb

```diff
@@ -175,126 +457,274 @@
-    "\n",
-    "    # Load social links directly\n",
-    "    social_links = {}\n",
-    "    social_links_file_path = member_json_dir / \"social_links.json\"\n",
+    "    edu_exp_df = edu_exp_df.sort_values(by='start_date', ascending=False)\n",
+    "    return edu_exp_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Function load social links directly"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 133,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def load_social_links(social_dir):\n",
+    "    \"\"\"\n",
+    "    Load social links from a JSON file.\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    social_dir : str or pathlib.Path\n",
+    "        The directory path containing the \"social_links.json\" file.\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    dict or None\n",
+    "        A dictionary containing social links or None if the file doesn't exist.\n",
+    "    \"\"\"\n",
+    "    social_links=None\n",
+    "    social_links_file_path = social_dir / \"social_links.json\"\n",
     "    if social_links_file_path.exists():\n",
     "        with open(social_links_file_path, \"r\") as f:\n",
     "            social_links = json.load(f)\n",
-    "\n",
-    "    # Load topmost project title\n",
-    "    current_project_title = None\n",
-    "    projects_file_path = member_json_dir / \"projects.json\"\n",
+    "    return social_links"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Function to load topmost project title"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 134,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def load_latest_project_title(project_dir):\n",
+    "    \"\"\"\n",
+    "    Load the title of the topmost project from a JSON file.\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    project_dir : str or pathlib.Path\n",
+    "        The directory path containing the \"projects.json\" file.\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    str or None\n",
+    "        The title of the topmost project or None if the file doesn't exist or is empty.\n",
+    "    \"\"\"\n",
+    "    projects_file_path = project_dir / \"projects.json\"\n",
+    "    topmost_project_title= None\n",
     "    if projects_file_path.exists():\n",
     "        projects_df = pd.read_json(projects_file_path)\n",
     "        if not projects_df.empty:\n",
-    "            current_project_title = projects_df.iloc[0].get(\"project_title\")\n",
-    "\n",
-    "    return education_experience_df, social_links, current_project_title"
+    "            topmost_project_title = projects_df.iloc[0].get(\"project_title\")\n",
```

Comment:
Can you put a comment saying why you need `iloc[0]`?

Reply:
Is this fine "# Fetching the project title from the first row of the DataFrame"?

Reply:
yes that is nice.